docs/requirements.txt

```diff
@@ -1,7 +1,7 @@
-sphinx
-sphinx_rtd_theme
-sphinx_copybutton
-sphinx_tabs
-pypandoc
-requests
-sphinx-autobuild
\ No newline at end of file
+Sphinx==3.3.1
```

Comment:
I don't think I'd care much about strict versioning here. I'd rather always use the latest available and if something breaks (rarely, if ever) we fix it in the code. It's not a critical part and is far less annoying than maintaining exact versions.

Reply:
I had a few problems with python3.7 installing old Sphinx v1. that is why ive done strict versioning here in `requirements.txt`

but i could change that if you want

Reply:
Late to the party, but you can also use comparison operators to get not just one specific version, but just "newer than 3.0": `Sphinx>=3.0.0`